yarn.lock

```diff
@@ -19357,6 +19365,13 @@ use-resize-observer@^5.0.0:
   dependencies:
     resize-observer-polyfill "^1.5.1"
 
+use-resize-observer@^6.0.0:
+  version "6.1.0"
+  resolved "https://registry.yarnpkg.com/use-resize-observer/-/use-resize-observer-6.1.0.tgz#d4d267a940dbf9c326da6042f8a4bb8c89d29729"
+  integrity sha512-SiPcWHiIQ1CnHmb6PxbYtygqiZXR0U9dNkkbpX9VYnlstUwF8+QqpUTrzh13pjPwcjMVGR+QIC+nvF5ujfFNng==
+  dependencies:
+    resize-observer-polyfill "^1.5.1"
+
```

Comment:
This is coming from Carbon. We use an older version. It would be very beneficial if we updated our version of this dep to also use v6.1.

I've created a separate issue https://github.com/IBM/carbon-addons-iot-react/issues/1250 but we can include here if you'd prefer @scottdickerson